### PR TITLE
Gizmos fixes

### PR DESCRIPTION
**Pull request by CedricGuillemet**
follow ups on:

https://forum.babylonjs.com/t/how-can-i-match-the-icons-direction-of-spotlights-lightgizmo-in-righthandedsystem/42929

https://forum.babylonjs.com/t/rotation-gizmo-sensitivity/42554

### CODE REVIEW

**Comment by bjsplat** [comment]
Please make sure to label your PR with "bug", "new feature" or "breaking change" label(s).
To prevent this PR from going to the changelog marked it with the "skip changelog" label.

**Comment by bjsplat** [comment]
Snapshot stored with reference name:
refs/pull/14118/merge

Test environment: 
https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14118/merge/index.html

To test a playground add it to the URL, for example:

https://babylonsnapshots.z22.web.core.windows.net/refs/pull/14118/merge/index.html#WGZLGJ#4600

Links to test babylon tools with this snapshot:

https://playground.babylonjs.com/?snapshot=refs/pull/14118/merge
https://sandbox.babylonjs.com/?snapshot=refs/pull/14118/merge
https://gui.babylonjs.com/?snapshot=refs/pull/14118/merge
https://nme.babylonjs.com/?snapshot=refs/pull/14118/merge

To test the snapshot in the playground with a playground ID add it after the snapshot query string:

https://playground.babylonjs.com/?snapshot=refs/pull/14118/merge#BCU1XR#0

**Review by RaananW — APPROVED** [review]

File: packages/dev/core/src/Gizmos/lightGizmo.ts [hunk 1]
```diff
@@ -138,7 +138,7 @@ export class LightGizmo extends Gizmo implements ILightGizmo {
             if ((light as any).direction) {
                 this.attachedMesh!.setDirection((light as any).direction);
                 this.attachedMesh!.computeWorldMatrix(true);
-                this._cachedForward.copyFrom(this.attachedMesh!.forward);
+                this._cachedForward.copyFrom(this.attachedMesh!.getScene().useRightHandedSystem ? this.attachedMesh!.forward.negate() : this.attachedMesh!.forward);
```
[the remark this answers is not in the document]
Comment (CedricGuillemet) [reply packages/dev/core/src/Gizmos/lightGizmo.ts]: fixed

File: packages/dev/core/src/Gizmos/lightGizmo.ts [hunk 2]
```diff
@@ -188,16 +188,20 @@ export class LightGizmo extends Gizmo implements ILightGizmo {
         }
         if ((this._light as any).direction) {
             // If the gizmo is moved update the light otherwise update the gizmo to match the light
-            if (Vector3.DistanceSquared(this.attachedMesh!.forward, this._cachedForward) > 0.0001) {
+            const forward = this.attachedMesh!.forward;
+            if (this.attachedMesh!.getScene().useRightHandedSystem) {
+                forward.negateInPlace();
```
[the remark this answers is not in the document]
Comment (CedricGuillemet) [reply packages/dev/core/src/Gizmos/lightGizmo.ts]: fixed

**Pull request merged**